### ISSUE TITLE
--dialects-export option doesn't work

### ISSUE THREAD

**Issue by Raipdy** [cmu-db/benchbase#252]
I tried --dialects-export option for tpcc and tpch with postgres, but it seems not work. Nothing is output on the console and no SQL file is created.

**Comment by XiN-1753827363** [comment]
> I tried --dialects-export option for tpcc and tpch with postgres, but it seems not work. Nothing is output on the console and no SQL file is created.

Hi, I met this issue too. Have you solved this problem yet?

**Comment by Raipdy** [comment]
Hello. I tried to look into the src code. But I couldn't find anything useful.



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年4月13日(星期四) 下午4:50
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [cmu-db/benchbase] --dialects-export option doesn&#39;t work (Issue #252)





  
I tried --dialects-export option for tpcc and tpch with postgres, but it seems not work. Nothing is output on the console and no SQL file is created.
  
Hi, I met this issue too. Have you solved this problem yet?

**Comment by Raipdy** [comment]
Yes. I just wanted to export the queries of transactions which had been executed.&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年4月13日(星期四) 晚上8:37
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [cmu-db/benchbase] --dialects-export option doesn&#39;t work (Issue #252)





  
Hello. I tried to look into the src code. But I couldn't find anything useful.
 …
  
So you met this problem in testing benchbase or oltpbenchmark?

**Comment by apavlo** [comment]
@Raipdy I don't think anybody has tested this feature in 10 years. If you have a fix please send a PR.
